python311Packages.simpful: 2.11.0 -> 2.11.1

fabaff opened the pull request:
Diff: https://github.com/aresio/simpful/compare/refs/tags/2.11.0...2.11.1

Changelog: https://github.com/aresio/simpful/releases/tag/2.11.1

## Description of changes

<!--
For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://nixos.org/manual/nixpkgs/unstable/#chap-reviewing-contributions
-->

pbsds commented on pkgs/development/python-modules/simpful/default.nix:
```diff
@@ -22,7 +22,7 @@ buildPythonPackage rec {
+    hash = "sha256-54WkKnPB3xA2CaOpmasqxgDoga3uAqoC1nOivytXmGY=";
   };
 
   # patch dated use of private matplotlib interface
```
The fix for this should be tagged in this release

fabaff replied:
Thanks, fixed.

fabaff committed:
python311Packages.simpful: 2.11.0 -> 2.11.1

Diff: aresio/simpful@refs/tags/2.11.0...2.11.1

Changelog: https://github.com/aresio/simpful/releases/tag/2.11.1

fabaff commented:
Result of `nixpkgs-review pr 268814` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>16 packages built:</summary>
  <ul>
    <li>python310Packages.aiortm</li>
    <li>python310Packages.aiortm.dist</li>
    <li>python310Packages.fuzzytm</li>
    <li>python310Packages.fuzzytm.dist</li>
    <li>python310Packages.pyfume</li>
    <li>python310Packages.pyfume.dist</li>
    <li>python310Packages.simpful</li>
    <li>python310Packages.simpful.dist</li>
    <li>python311Packages.aiortm</li>
    <li>python311Packages.aiortm.dist</li>
    <li>python311Packages.fuzzytm</li>
    <li>python311Packages.fuzzytm.dist</li>
    <li>python311Packages.pyfume</li>
    <li>python311Packages.pyfume.dist</li>
    <li>python311Packages.simpful</li>
    <li>python311Packages.simpful.dist</li>
  </ul>
</details>
